src/packages/cli/src/initialize/ethereum.ts

```diff
@@ -6,41 +6,43 @@ export default function (provider: Provider, cliSettings: CliSettings) {
   const liveOptions = provider.getOptions();
   const accounts = provider.getInitialAccounts();
 
-  console.log("");
-  console.log("Available Accounts");
-  console.log("==================");
-
   const addresses = Object.keys(accounts);
-  const ethInWei = 1000000000000000000n;
-
-  addresses.forEach(function (address, index) {
-    const balance = accounts[address].balance;
-    const strBalance = balance / ethInWei;
-    const about = balance % ethInWei === 0n ? "" : "~";
-    let line = `(${index}) ${toChecksumAddress(
-      address
-    )} (${about}${strBalance} ETH)`;
-
-    if (!accounts[address].unlocked) {
-      line += " 🔒";
-    }
+  if (addresses.length > 0) {
```

Comment:
Would it be better to still print this information? Like tell them they have no accounts unlocked? It could be something like
```
Available Accounts
==============
(no accounts unlocked)
```

What do you think?

Reply:
I only did it this way because we already omit sections in a similar fashion. But doing what you are suggesting seems more helpful. :+1: